### DISCUSSION TITLE
Configuring Thumbnail Generation for WYSIWYG Editor Drag and Drop

### DISCUSSION THREAD

**Discussion by pdchaudhary** [pimcore/pimcore#16175] [Q&A / Support]
I am currently working with the WYSIWYG editor in Pimcore. When images are dragged and dropped into the editor, thumbnails are automatically generated for these images. I'm interested in finding out if there is a way to apply a specific thumbnail configuration for this process instead of relying on the default settings.

Specifically, is it possible to set a custom thumbnail name so that the generated thumbnail is based on predefined settings?

For example:

URL with Auto-Generated Thumbnail: image-thumb__{assetId} __auto_613546775741237bc63c12d3aecc8038/test.fd9bafd1.png

URL with Specific Thumbnail: image-thumb__{assetId}__{thumbnail_name}/test.fd9bafd1.png

I would really appreciate any guidance or suggestions on this matter!

Thank you in advance

**Comment by NiklasBr** [comment]
Unfortunately: [#5574 [Data objects] Thumbnail transformations for images in wysiwyg fields ](https://github.com/pimcore/pimcore/issues/5574)
